modules/medianetBidAdapter.js

```diff
@@ -162,6 +162,15 @@ function slotParams(bidRequest) {
   };
   let bannerSizes = utils.deepAccess(bidRequest, 'mediaTypes.banner.sizes') || bidRequest.sizes || [];
 
+  const videoInMediaType = utils.deepAccess(bidRequest, 'mediaTypes.video') || {};
+  const videoInParams = utils.deepAccess(bidRequest, 'params.video') || {};
+  let videoCombinedObj = Object.assign({}, videoInMediaType);
```

Comment:
Can't we do something like this?

```(javascript)
Object.assign({}, videoMediaType, videoInParams);
```

Reply:
yes, this makes sense. I have incorporated the changes.